ccstake/README.md

```diff
@@ -0,0 +1,207 @@
+
+```bash
+$> ccstake show
+```
+
+You can see specific account's CCS state by passing an address to the `show` command.
+
+```bash
+$> ccstake show cccqxyyc4yu3pc2pzl2y0tec26qxau3a27lq5ntee9j
+```
+
+#### transfer
+
+To transfer CCS to someone from an account, you can use `transfer` command.
+
+```bash
+$> ccstake transfer \
+     --from cccqxyyc4yu3pc2pzl2y0tec26qxau3a27lq5ntee9j \
+     --to cccq9h7vnl68frvqapzv3tujrxtxtwqdnxw6y4u3qm5 \
+     --quantity 10000 \
+     --fee 10
+```
+
+#### delegate
+
+To delegate CCS to someone from an account, you can use `delegate` command.
+
+```bash
+$> ccstake delegate \
+     --from cccqxyyc4yu3pc2pzl2y0tec26qxau3a27lq5ntee9j \
+     --to cccq9h7vnl68frvqapzv3tujrxtxtwqdnxw6y4u3qm5 \
+     --quantity 10000 \
+     --fee 10
+```
+
+#### revoke
+
+To revoke delegated CCS to a delegatee from a delegator, you can use `revoke` command.
+
+```bash
+$> ccstake revoke \
+     --delegator cccqxyyc4yu3pc2pzl2y0tec26qxau3a27lq5ntee9j \
+     --delegatee cccq9h7vnl68frvqapzv3tujrxtxtwqdnxw6y4u3qm5 \
+     --quantity 10000 \
+     --fee 10
+```
+
+#### batch-delegate
+
+To manage multiple delegations to validators across the multiple stakeholder accounts, you can use `batch-delegate` command. To use it, you need a distribution file, and password file.
+
+The distribution file is a json file similar to this:
+```json
+{
+  "stakeholders": [
+    "cccq9qwg08jnn4agnaex3pty5hcq04m2h87ryxh9p5q",
+    "cccqyuzgh3y8w3xtrzdfrjs6yk6wrhh30y6gys2nv6l",
+  ],
+  "fee": 10,
+  "distributions": [
+    {
+      "validator": "cccq98jmz9muznaun3xhtmumt7txx8d4ehdlcn5v3hz",
+      "quantity": 10000
+    },
+    {
+      "validator": "cccqyyk336h4d5ddv20h6hhdh35u6r7j5dn7chl2xaz",
+      "quantity": 20000
+    },
+    {
+      "validator": "cccq8hekjzqhpcha528jalj2qyjhd5849kpxgrhfc76",
+      "quantity": 30000
+    },
+  ]
+}
+```
+
+The password file is similar to this:
+```json
+{
+    { "address": "cccq9qwg08jnn4agnaex3pty5hcq04m2h87ryxh9p5q", password: "super-strong-password" },
+    { "address": "cccqyuzgh3y8w3xtrzdfrjs6yk6wrhh30y6gys2nv6l", password: "very-strong" }
+}
+```
+The password file should contain all passwords of all stakeholders in the distribution file.
+
+When these file is prepared, you can dry-run to see the planned transactions to distribute accordingly.
+
+```bash
+$> ccstake batch-delegate ./distribution.json \
+     --password-path=./passwords.json \
+     --dry-run
+```
+
+If you are satisfied with the plan, you can go ahead with the following command.
+
+```bash
+$> ccstake batch-delegate ./distribution.json \
+     --password-path=./passwords.json
+```
+
+However, there might be a changes in the plan when the overall situation is changed, and the transactions will not run in atomically. When some of the transactions in the plan is failed, succeeded transactions will not be reverted.
+
+### Governance
+
+In the code chain, stakeholder governance is reflected by changing the parameters of the chain.
```

Comment:
In CodeChain, the stakeholder governance is reflected by changing the parameters of the chain.